lib/assets/logo/Icon

```diff
@@ -0,0 +1 @@
+
```

Comment:
Not sure why it is empty possible corruption on this one, but i can see the logo in the other commit so it should be ok to remove this.